pkg/commands/edit/add/all.go

```diff
@@ -30,10 +30,10 @@ func NewCmdAdd(fsys fs.FileSystem, v ifc.Validator, kf ifc.KunstructuredFactory)
 		Long:  "",
 		Example: `
 	# Adds a secret to the kustomization file
-	kustomize edit add secret NAME --from-literal=k=v
+	kustomize edit add secret NAME --plugin-name=literals --args=k=v
```

Comment:
Since you're modifying this code, suggest that we just interpret all the args after the configmap/secret name to be arguments to the generator.

I.e., no `--args` flag.  Just take the args from the command in order.